README.md

```diff
@@ -13,63 +13,19 @@ for implementation details.
 In order to use this server, you should have:
 
 - This project checked out. You may wish to create a GitHub Fork of the project and check that out instead so that you can customize the project and save the results to GitHub.
-
-### and either
- - Oracle Java (JDK) installed: Minimum JDK8 or newer.
- - Apache Maven build tool (newest version)
-
-### or
- - Docker, as the entire project can be built using multistage docker (with both JDK and maven wrapped in docker).
-
-## Running locally
-
-The easiest way to run this server entirely depends on your environment requirements. At least, the following 4 ways are supported:
-
-### Using jetty
-```bash
-SPRING_PROFILES_ACTIVE="local" mvn -Djetty.port=8888 jetty:run
-```
-
-Server will then be accessible at http://localhost:8888/ and eg. http://localhost:8888/fhir/metadata.
-
-### Using Spring Boot
-
-This runs with Tomcat.
-
-```bash
-mvn clean package spring-boot:repackage -Pboot && java -jar target/ROOT.war
-```
-Server will then be accessible at http://localhost:8080/ and eg. http://localhost:8080/fhir/metadata. Remember to adjust you overlay configuration in the application.yaml to eg.
-
-```yaml
-    tester:
-      -
-          id: home
-          name: Local Tester
-          server_address: 'http://localhost:8080/fhir'
-          refuse_to_fetch_third_party_urls: false
-          fhir_version: R4
-```
+- Docker, as the entire project can be built using multistage docker (with both JDK and maven wrapped in docker).
 
 ### Using Docker
 
-You can use the docker image generated by:
-
-Build the docker image:
+You can use the `docker-compose` to run the app and DB servers by executing the following command:
 
 ```shell
-$ docker build -t fhir-server .
+$ docker-compose up --build
 ```
 
 Run the server from the docker image, on a specific Spring profile (in this case, `application-staging.yaml`), with environment variables being passed to the docker container:
 
-```shell
-$ docker run -it -e DB_URL="jdbc:postgresql://host.docker.internal:5432/db?currentSchema=hapi_fhir" -e DB_USERNAME="admin" -e DB_PASSWORD="admin" -e SPRING_PROFILES_ACTIVE="staging" fhir-server
-```
-
-`host.docker.internal` is a reference to the host computer running the docker container (likely your computer).
-
-Update environment variables and their values accordingly.
```

Comment:
This too, as its the way to run it on a local Docker image similar to what we have on the cloud.

Reply:
https://github.com/metriport/fhir-server/pull/7#discussion_r1122156133